integreat_cms/cms/constants/machine_translation_budget.py

```diff
@@ -0,0 +1,27 @@
+"""
+This module contains the available choices for machine translation budget
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Final
+
+
+MINIMAL: Final = 50000
+XS: Final = 100000
+S: Final = 150000
+M: Final = 250000
+L: Final = 1000000
+XL: Final = 5000000
```

Comment:
```suggestion
HUGE: Final = 5000000
```

I think we should rename this constants for three reasons: a. At the moment I don't find "L" very readable (for example in the migration file) and it's going to be difficult to be found in the code later on. b. apart from the readability I think it makes it easier for us to add new sizes in the future (for example ENORMOUS for bigger than huge or SLIM for a size between small and medium and c. just out of consistency, because we already use MINIMAL for one of the sizes and I think it'd be good if we stick to one convention only :)